### DISCUSSION TITLE
Is it possible to select which graphics card to use?

### DISCUSSION THREAD

**Discussion by Teedo145** [lllyasviel/Fooocus#2098] [Q&A] [answered]
I'm planning on putting 2 graphics cards in my pc and I want to run 2 instances of fooocus. Is it possible to run one on each graphics card?

**Comment by mashb1t** [comment, marked as answer]
This is possible using `--gpu-device-id DEVICE_ID` in your startup args. Please be advised that your GPUs will need to have at least both 8GB (best 12GB) of VRAM (Nvidia) and probably more (AMD) to effectively work in parallel so your RAM/Drive/CPU aren't the bottlenecks.
